splunk_otel/metrics.py

```diff
@@ -44,13 +44,28 @@ def start_metrics() -> MeterProvider:
 
 def _configure_metrics() -> MeterProvider:
     # pylint: disable=import-outside-toplevel
-    from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import (
-        OTLPMetricExporter,
-    )
+    # FIXME yes, this doesn't match up to the spec in terms of looking for
+    # relevant environment variables, etc., but this is a stopgap because a lot
+    # of this initialization sequence needs to be burned down and replaced with the
+    # upstream SDK's Configurator.
+    metrics_exporter = None
+    try:
+        from opentelemetry.exporter.otlp.proto.grpc.metric_exporter import (
+            OTLPMetricExporter,
+        )
+
+        metrics_exporter = OTLPMetricExporter()
+    except Exception:  # pylint:disable=broad-except
+        logger.info("Cannot load grpc metrics exporter, trying http/protobuf")
```

Comment:
[This comment](https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/python/requirements.txt#L2-L3) in the operator repo seems relevant.